includes/widgets/image-box.php

```diff
@@ -755,8 +755,11 @@ protected function register_controls() {
 	protected function render() {
 		$settings = $this->get_settings_for_display();
 
+		$text_description = wp_kses_post( $settings['description_text'] );
+		$text_title = wp_kses_post( $settings['title_text'] );
```

Comment:
Maybe name it 
```suggestion
		$description_text = wp_kses_post( $settings['description_text'] );
		$title_text = wp_kses_post( $settings['title_text'] );
```
like the settings keys